public/javascripts/items.js

```diff
@@ -74,7 +74,6 @@ const completedClicked = event => {
   const element = $( event.target )
   const id = element.data( 'id' )
   const completed = ! element.data( 'completed' )
-
   fetch( `/items/${id}`, params({ completed: completed } ) )
```

Comment:
Please maintain whitespace between logical blocks - there should be a separation between the setup and logic here.